### PR TITLE
Case insensitive lookup for SortedVectorKP

### PR DESCRIPTION
**Pull request by mlove-au**
I was thinking of having two overloads in the comparator vs a template, since the implementation assumes either std::string or boost::string_ref types, but it would be identical code and I doubt you are going to add another type for the keys.

### CODE REVIEW

**Comment by mlove-au** [comment]
oh, do you want to bump the version number to 0.8.0 if this is good? Since it also requires c++14 it should go up for sure. but I'm not sure if that should be 1.0.0,? I can drop the c++14 if you like, but it adds back the horrible decltypes

**Comment by daedric** [comment]
0.8.0 is fine. The interface stays compatible

**Comment by daedric** [comment]
Please change the version and I'll merge it right away :)
Thanks !

**Comment by daedric** [comment]
and the readme :)

File: include/httpp/utils/SortedVectorKP.hpp [hunk 1]
```diff
@@ -13,25 +13,45 @@
 
 # include <algorithm>
 # include <vector>
+# include <strings.h>
```
Comment (daedric) [inline include/httpp/utils/SortedVectorKP.hpp]: `#include <cstring>`

**Pull request merged**